addons/core/addon-test-support/setup.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  AnnotationsRecord,
+  ConvertersOpinionatedRecord,
+  LabelOverridesRecord,
+  StepImplementationsRecord,
+} from './types';
+
+export function setupAnnotations(annotations: AnnotationsRecord): void {
+  // @todo
+}
+
+export function setupDictionary(dictionary: ConvertersOpinionatedRecord): void {
+  // @todo
+}
+
+export function setupLabelOverrides(labelOverrides: LabelOverridesRecord): void {
+  // @todo
+}
+
+export function setupSteps(steps: StepImplementationsRecord): void {
```

Comment:
Hmm, I would rather put these in corresponding files in `core`: `dictionary.ts`, `annotations.ts`, etc.

They all could share the same tooling for store.

Reply:
Ok, I can do that. But at the end this is a private implementation detail. You shouldn't import from `/setup` anyway. 

The important piece is the public API we are now exposing. Are you d'accord with that?